src/bank.rs

```diff
@@ -602,7 +602,7 @@ impl Bank {
         for (tick_height, stake) in ticks_and_stakes.iter() {
             if ((current_tick_height - tick_height) as usize) < MAX_ENTRY_IDS {
                 total += stake;
-                if total > supermajority_stake {
+                if total >= supermajority_stake {
```

Comment:
won't this return success for 66% of the total stake?

Reply:
Yes, this was just a workaround, trying to fix it the right way.

Reply:
👍